src/pnal.h

```diff
@@ -33,9 +33,9 @@ extern "C" {
 #define PNAL_MAKEU16(a, b)                                                     \
    (((uint16_t) ((a)&0xff) << 8) | (uint16_t) ((b)&0xff))
 
-#define PNAL_INET_ADDRSTRLEN 16
-#define PNAL_ETH_ADDRSTRLEN  18
-#define PNAL_HOST_NAME_MAX   64 /* Value from Linux */
+#define PNAL_INET_ADDRSTRLEN 16 /** Incl termination */
+#define PNAL_ETH_ADDRSTRLEN  18 /** Incl termination */
+#define PNAL_HOST_NAME_MAX   64 /** Incl termination. Value from Linux. */
```

Comment:
Could this value be defined in pnal_sys.h instead, with a default in pnal.h?

Reply:
No problem I will do that. (Why do we need a default value? For example OS_BUF_MAX_SIZE is defined in pnal_sys.h without default value.

Reply:
Just for convenience so the port can optionally override the default. It might not make sense in this case.

Reply:
Done